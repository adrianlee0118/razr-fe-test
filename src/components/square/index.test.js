import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Square from "./index";

describe("Square", () => {
  //Component test
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Square />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  //Snapshot test
  test("has a valid snapshot", () => {
    const component = renderer.create(<Square />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
