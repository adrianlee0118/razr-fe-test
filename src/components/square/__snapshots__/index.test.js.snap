// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square has a valid snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "display": "inline-block",
      "height": undefined,
      "margin": 20,
      "padding": 10,
      "transform": "rotate(undefineddeg)",
      "width": undefined,
    }
  }
/>
`;
